website_oca_integrator/demo/res_users.xml

```diff
@@ -15,7 +15,7 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
         <field name="street">Palermo, Capital Federal</field>
         <field name="street2">C1414CMS Capital Federal</field>
         <field name="image" type="base64"
-               file="base/static/src/img/avatar.png"/>
+               file="base/static/img/avatar.png"/>
```

Comment:
Why changing this path?

Reply:
the path have changed so this cause error

Reply:
But the other path seems more correct.

Reply:
Do you want to open a PR to odoo with this suggestion?

Reply:
Oh, I didn't see that it was part of Odoo core. Let's keep it this way then. I said it was more correct because odoo previously uses that path. Maybe they have changed their common structure...